ebr_board/ebr_board.py

```diff
@@ -77,10 +80,46 @@ def configure_api():
     ebr_board_api.init_app(api_bp)
 
 
+def lambda_handler(event, context):
+    """
+    Set this as the target for AWS Lambda
+    """
+    import awsgi
+    import os
+    from ssm_parameter_store import EC2ParameterStore
+
+    config_name = os.environ.get("config_name", "ebr_board_config")
+    vault_config_name = os.environ.get("vault_config_name", "ebr_board_vault_config")
+    vault_creds_name = os.environ.get("vault_creds_name", "ebr_board_vault_creds")
+    config_format = os.environ.get("config_format", "yaml")
+
+    elastic_cert_name = os.environ.get("elastic_cert_name", None)
+    vault_cert_name = os.environ.get("vault_cert_name", None)
+
+    store = EC2ParameterStore()
+    config = store.get_parameters([config_name, vault_config_name, vault_creds_name], decrypt=True)
+
+    # Get certs that are stored in the ssm parameter store rather than Vault
+    if elastic_cert_name:
+        elastic_cert = store.get_parameter(elastic_cert_name)[elastic_cert_name]
+        with open("/tmp/elastic.crt", "w") as elastic_cert_file:
+            elastic_cert_file.write(elastic_cert)
+
+    if vault_cert_name:
+        vault_cert = store.get_parameter(vault_cert_name)[vault_cert_name]
+        with open("/tmp/vault.crt", "w") as vault_cert_file:
```

Comment:
Can't we just pass a string instead of writing the content of the certificate into a file?

Reply:
I will double check if this is possible, but with almost all Python libraries it is not, due to their usage of the OpenSSL library, which also doesn't support it.

Reply:
Actually, its looking like maybe I can force it in, at least for elasticsearch. Trying that out now.